exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,18 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  } else return false;
+}
 
```

Comment:
plz. think about the type and value of the expression number < 0, and how you can simplify this function